### ISSUE TITLE
[🐛 Bug]: Wont allow me to download OAuth 2.0 credentials

### ISSUE THREAD

**Issue by CypherCrypts** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#953]
### What went wrong?

it says that Reached limit on number of clients in this project. Try in a new project.

![Screenshot 2024-10-21 212806](https://github.com/user-attachments/assets/71bfa184-7938-43de-8614-e2bef82d8ac9)

### What did you expect to happen?

my OAuth 2.0 credentials to download

### Steps to reproduce the bug

1. Go to https://console.actions.google.com/project/echo-2--20057/deviceregistration/
2. Download 0Auth 2.0 credentials of your project

### App Version

Built from source

### OS Version

Windows 11 Home

### Pre-submission Checklist

- [X] I have checked [the issue tracker](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues?q=is%3Aissue+label%3A%22%F0%9F%90%9B+bug%22) and didn't find any duplicate issue
- [X] I have checked the [Frequently Asked Questions (FAQs)](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)) and can confirm that there's no fixes or workarounds for this issue
- [X] This issue only covers one particular bug or issue

**Comment by mayumidesu** [comment]
have the same problem :(
![image](https://github.com/user-attachments/assets/14fdba9b-073f-4fab-a835-d3ddaf835750)

**Comment by im-coder-lg** [comment]
Have you tried to create another project and try it again? This behaviour is unusually odd, and I've never seen this till now.

**Comment by Ibrahim6788** [comment]
im having the same issue, ive tried 3 diffrent projects with different names, regions, and device types and still shows up with this "reached limit on number of clients in this proect, Try in a new project." error.

**Comment by im-coder-lg** [comment]
I'll try it, it's been a long time actually.

**Comment by CypherCrypts** [comment]
> Have you tried to create another project and try it again? This behaviour is unusually odd, and I've never seen this till now.

Yes, yes i have, like 2 or 3 times more and then i gave up cuz it didnt work

**Comment by Shah-do** [comment]
same issue as me

**Comment by Brijesh2772** [comment]
hi brother it worked for me so basically i added a Oauth client id in api credentials tab with desktop client and it worked
And then the same process.
download the json and enter the json in app and relaunch it will open chrome page and enter ur account and get the code from the url it works 
![Screenshot 2024-11-08 133731](https://github.com/user-attachments/assets/b3ad33d7-e7f2-4b58-ab51-793cc8f10383)

**Comment by Adr1an1002** [comment]
> Brijesh2772

How did you do that? can you explain it?

**Comment by Adr1an1002** [comment]
@Brijesh2772 I did that but then I got this error #881

**Comment by jjarongay50** [comment]
> hi brother it worked for me so basically i added a Oauth client id in api credentials tab with desktop client and it worked And then the same process. download the json and enter the json in app and relaunch it will open chrome page and enter ur account and get the code from the url it works ![Screenshot 2024-11-08 133731](https://private-user-images.githubusercontent.com/115490513/384279711-b3ad33d7-e7f2-4b58-ab51-793cc8f10383.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE2MjE1OTksIm5iZiI6MTczMTYyMTI5OSwicGF0aCI6Ii8xMTU0OTA1MTMvMzg0Mjc5NzExLWIzYWQzM2Q3LWU3ZjItNGI1OC1hYjUxLTc5M2NjOGYxMDM4My5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExNFQyMTU0NTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iOWE1Nzg4YmMxMDZkZmFkZmZiOWQ1N2IwMDdjOWEyYmQwZjM1Mzk5NzdjNjVmNWVlOTAzZDRmYzFkY2VkMmMzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.kNFmat6nljV64h0HJlhyO77t8oEmbkTG8bvvCmpGLTg)

It worked on me! Thank you Brijesh

**Comment by YTpooshistic** [comment]
> hi brother it worked for me so basically i added a Oauth client id in api credentials tab with desktop client and it worked And then the same process. download the json and enter the json in app and relaunch it will open chrome page and enter ur account and get the code from the url it works ![Screenshot 2024-11-08 133731](https://private-user-images.githubusercontent.com/115490513/384279711-b3ad33d7-e7f2-4b58-ab51-793cc8f10383.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI1NTE4OTksIm5iZiI6MTczMjU1MTU5OSwicGF0aCI6Ii8xMTU0OTA1MTMvMzg0Mjc5NzExLWIzYWQzM2Q3LWU3ZjItNGI1OC1hYjUxLTc5M2NjOGYxMDM4My5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyNVQxNjE5NTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05OWY5MmEwMmQ1MTk2ODJjMjNmM2VlYWUwODYxZGNkMDViN2U2NDYxZWU1NTE1MWU5ZjY5NTdlMjEzOTZkMjE3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.2M-AsDSXxeRjeCaPa_rL4mjSe_G-TIQ6HQocYIth_8E)I

I can't get token. App opens web peg with log in, I log in and when I must see a token I see localhost page
![Снимок экрана от 2024-11-25 19-24-23](https://github.com/user-attachments/assets/32447bdd-ac02-46f1-9dfa-531f92b3cdf5)

**Comment by Diode-exe** [comment]
You have to make the project in a new account. That's the only thing that fixed it for me, and then add your main account that failed into the OAuth credentials screen.

**Comment by Jacobisfunnylol** [comment]
#how

**Comment by Jacobisfunnylol** [comment]
where is API Credentials [Brijesh2772](https://github.com/Brijesh2772)?
![image](https://github.com/user-attachments/assets/bfb09baf-4011-44a7-b89a-75ad624129c1)

**Comment by Jeff77887788** [comment]
> where is API Credentials [Brijesh2772](https://github.com/Brijesh2772)? ![image](https://private-user-images.githubusercontent.com/147019108/400185534-bfb09baf-4011-44a7-b89a-75ad624129c1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY1NzU2OTYsIm5iZiI6MTczNjU3NTM5NiwicGF0aCI6Ii8xNDcwMTkxMDgvNDAwMTg1NTM0LWJmYjA5YmFmLTQwMTEtNDRhNy1iODlhLTc1YWQ2MjQxMjljMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDExMVQwNjAzMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zYzc2NDkxOWJkNTgyMjc3NTVjMGU5MzA4MjgyNWNjMmVkMmM5Y2QxZTY3MDNmN2MxMzRiNTBiNTIwMGM4MzdjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.-bs7-v4MLDUFM9XXoBgsPa8eDo-9LmBWp2ub7mgjWTQ)

(https://console.cloud.google.com/apis/credentials) Follow this link, under oAuth consent screen, press create and follow the steps.

**Comment by Diode-exe** [comment]
@YTpooshistic You see where it says ?=code in the URL bar? That's the code you need. Copy everything past ?=code and paste it where it asks for the code.
